src/steps/sse.ts

```diff
@@ -103,12 +106,35 @@ export default async function (
       }
 
       resolve(true)
+    }
+
+    const timeout = setTimeout(() => {
+      console.debug('SSE timed out')
```

Comment:
No debug please, return an error if you this should be visible in the CLI

Reply:
Timeout was considered a test pass, I didn't want to change the behavior (see https://github.com/stepci/runner/pull/121#discussion_r1636879494). How should I log this then?

Reply:
If it times out and no message was received during the time, the test should be marked as failed

Reply:
No message at all or not all expected messages if applicable? (I have my idea, I just want to follow yours)

Reply:
yes

Reply:
Which one? 😅

Reply:
If not every expected message matched or no messages matched and a timeout occurs, the checks for them should fail - see current behaviour.